lib/std/Progress.zig

```diff
@@ -246,7 +246,7 @@ pub const Node = struct {
         }
         const index = n.index.unwrap() orelse return;
         const parent_ptr = parentByIndex(index);
-        if (parent_ptr.unwrap()) |parent_index| {
+        if (@atomicLoad(Parent, parent_ptr, .acquire).unwrap()) |parent_index| {
```

Comment:
I added the acquire-load to match the other acquire-load of `parentByIndex()` in `Node.init()`. I think it makes sense, but tbh I'm not 100% sure it's necessary.